pycbc/libutils.py

```diff
@@ -165,12 +166,21 @@ def get_ctypes_library(libname, packages, mode=None):
         libdirs += pkg_config_libdirs(packages)
     except ValueError:
         pass
-    # Next, if we are in a virtual environment, search inside its '/lib'
-    if "VIRTUAL_ENV" in os.environ:
-        libdirs.append(os.path.join(os.environ["VIRTUAL_ENV"], "lib"))
-
-    # Note that the function below can accept an empty list for libdirs, in which case
-    # it will return None
+    # We might be using conda/pip/virtualenv or some combination. This can
+    # leave lib files in a number of directories that LD_LIBRARY_PATH or
+    # pkg_config can miss.
+    # This should work, but the unittests use a venv within a venv type thing
+    # libdirs.append(os.path.join(sys.prefix, "lib"))
+    # So instead try something more hacky ...
+    for path in sys.path:
+        # Don't know how to make this work also in Windows
```

Comment:
@spxiwh We don't support windows atm anyway. Some of our dependencies don't support it at all right now (at least in released versions).